auraed/src/runtime/error.rs

```diff
@@ -29,14 +28,13 @@ impl From<RuntimeError> for Status {
         let msg = err.to_string();
         error!("{msg}");
         match err {
-            RuntimeError::MissingArgument { arg: _ } => {
-                //| RuntimeError::ValidationError { 0: _ } => {
+            RuntimeError::MissingArgument { .. } => {
                 Self::failed_precondition(msg)
             }
-            RuntimeError::Internal { msg: _, err: _ }
-            | RuntimeError::CgroupsError { 0: _ }
-            | RuntimeError::Other { 0: _ } => Self::internal(msg),
-            RuntimeError::Unallocated { resource: _ } => Self::not_found(msg),
+            RuntimeError::Internal { .. }
+            | RuntimeError::CellsError { .. }
```

Comment:
is it true that all CellsErrors are internal?  i expected that we might have different outer "public" errors coming from different methods within each submodule so having this top-level runtime error was the right way to determine the mapping from one set of errors to the statuses.

Reply:
I expect you are right. We will need to tackle mapping them. My suggestion would be to define the errors better (i.e., CellExits, CellNotFound, etc.) and then that would be easier to map to Status correctly